srcDoc/release_notes.rst

```diff
@@ -98,6 +98,15 @@ Added 2 new Governance profiles to the list of generated governance files in
 
 As well as their respective signed versions (`signed_`...).
 
+Added news print formats for data display (#280)
```

Comment:
New output formats available to display data (#280)

Reply:
ok